packages/editor/src/plugins/fill-in-the-blanks-exercise/index.tsx

```diff
@@ -45,8 +46,7 @@ function createState() {
       },
     }),
     mode: string(defaultMode),
-    // This could be used to store additional answers for mode drag-and-drop in the future
-    // additionalDraggableAnswers: optional(list(object({ answer: string() }))),
+    additionalDraggableAnswers: list(object({ answer: string() })),
```

Comment:
```suggestion
    extraDraggableAnswers: list(object({ answer: string() })),
```

I'd like this naming better, since I read additional as "more of the same" and extra as "more of something special".